frontend/public/components/namespace.jsx

```diff
@@ -275,7 +278,13 @@ const ProjectLink = connect(null, { setActiveNamespace: UIActions.setActiveNames
         title={project.metadata.name}
         type="button"
         className="co-resource-item__resource-name"
-        onClick={() => setActiveNamespace(project.metadata.name)}
+        onClick={() => {
+          setActiveNamespace(project.metadata.name);
+          removeQueryArgument('project-name');
+          store.dispatch(
+            k8sActions.filterList(referenceForModel(ProjectModel), 'project-name', ''),
+          );
```

Comment:
Why is this necessary? 

Also, `store` is accessed globally instead of through `connect`,

Reply:
@christianvogt When I search and select a project from the table links and then go back to `all-projects` from the namespace dropdown, the URL has no sign of project-name` query param, also the input filter is empty but the ListPageWrapper_ component receives filters: {project-name : <abc>} from the firehose. So this code change takes care of that, I am unsure if the bug fix for https://issues.redhat.com/browse/ODC-3896 should take care of this. Let me know your views

Reply:
I feel like this is a patch for the real problem lurking with https://issues.redhat.com/browse/ODC-3896
Maybe if we keep it at least we need a comment.

Regardless we need to avoid using the redux store as a global and instead use connect.